src/sentry/tsdb/base.py

```diff
@@ -90,6 +90,8 @@ class TSDBModel(Enum):
     project_total_received_cors = 609
     # the number of events filtered because their group was discarded
     project_total_received_discarded = 610
+    # the number of events filtered because they refer to a healthcheck endpoint
+    project_total_healthcheck = 611
```

Comment:
do we need this? Jan told me once that this is something really old and not very used

Reply:
just saw that it is being used. so never mind